src/EnergyPlus/DataContaminantBalance.hh

```diff
@@ -288,8 +231,7 @@ struct ContaminantBalanceData : BaseGlobalStruct
 
     Array1D<Real64> CONTRAT;           // Zone CO2 at the previous time step used in Exact and Euler method
     Array1D<Real64> MixingMassFlowCO2; // Mixing MASS FLOW * CO2
-    int NumContControlledZones = 0;
-    Real64 OutdoorCO2 = 0.0; // Outdoor CO2 level
```

Comment:
I'd love to go through this and clean up all the Array1D's, but I'm not going to add that to this PR.